src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoGenerator {
+    private static List<LottoNumber> numbers = generateNumber();
+
+    public List<Lotto> autoPublish(int buyingCount) {
+        List<Lotto> lottos = new ArrayList<>();
+
+        for (int i = 0; i < buyingCount; i++) {
+            lottos.add(pickNumber());
+        }
+        return lottos;
+    }
+
+    private static List<LottoNumber> generateNumber() {
+        numbers = new ArrayList<>();
+
+        for (int i = LottoNumber.MIN_NUMBER; i <= LottoNumber.MAX_NUMBER; i++) {
+            numbers.add(new LottoNumber(i));
+        }
+        return numbers;
+    }
+
+    private Lotto pickNumber() {
+        Collections.shuffle(numbers);
+        List<LottoNumber> lottoNumbers = new ArrayList<>();
+
+        for (int i = 0; i < Lotto.PICK_COUNT; i++) {
+            lottoNumbers.add(numbers.get(i));
+        }
+        Collections.sort(lottoNumbers);
+
+        return new Lotto(lottoNumbers);
```

Comment:
이렇게 람다식으로 표현해 볼 수 있을 것 같습니다 😄 
이 부분은 참고만 부탁 드리겠습니다 🙇 

```suggestion
        return numbers.stream()
                .limit(Lotto.PICK_COUNT)
                .sorted()
                .collect(Collectors.collectingAndThen(Collectors.toList(), 
                        Lotto::new));
```